### PR TITLE
Save influx offsets to streamsets

### PR DESCRIPTION
**Pull request by AntonZelenin**

### CODE REVIEW

File: agent/src/agent/pipeline/config/handlers/influx.py [hunk 1]
```diff
@@ -156,28 +94,15 @@ def update_source_configs(self):
                     'dimensions': columns,
                     'metric': measurement_name,
                     'delay': delay,
-                    'interval': str(interval) + 's',
+                    # interval is in minutes
```
Comment (dariakharlan) [inline agent/src/agent/pipeline/config/handlers/influx.py:97]: why interval in minutes? don't change any config to not break backward compatibility

File: agent/src/agent/pipeline/config/jython_scripts/influx.py [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+try:
+    sdc.importLock()
+    import time
+    from datetime import datetime, timedelta
+finally:
+    sdc.importUnlock()
+
+
+def get_now_with_delay():
+    return int(time.time()) - int(sdc.userParams['INTERVAL_IN_MINUTES']) * 60
```
Comment (dariakharlan) [inline agent/src/agent/pipeline/config/jython_scripts/influx.py:12]: I don't see any delay here

**Review by dariakharlan — CHANGES_REQUESTED** [review]

File: agent/src/agent/pipeline/config/handlers/influx.py (hunk 1 above)
Comment (AntonZelenin) [reply agent/src/agent/pipeline/config/handlers/influx.py:97]: omg, I thought it was in minutes

File: agent/src/agent/pipeline/config/jython_scripts/influx.py (hunk 2 above)
Comment (AntonZelenin) [reply agent/src/agent/pipeline/config/jython_scripts/influx.py]: oops

**Review by dariakharlan — APPROVED** [review]

**Pull request merged**